openedx/edx-ora2
# fix: increase item_id max lenght from 128 to 255 for peer, staff and studenttraining workflows

**TL;DR -** [ A short summary of what this PR does and why ]

The current max length for item_id fields of peerworkflow, staffworkflow and studenttrainingworkflow is 128 characters which are insufficient for course_id with a length greater than 70 characters. This PR alters the max length of item_id from 128 to 255.
[Related forum post](https://discuss.openedx.org/t/report-open-response-edx-ora2-bug-cannot-submit-responses-for-courses-with-ids-greater-than-70-characters/12368/2)

**What changed?**

- Increase item_id max length of peerworkflow, staffworkflow and studenttrainingworkflow from 128 to 255.

**Developer Checklist**


**Testing Instructions**

1. Check if the definitions of these fields are varchar(255). openedx.assessment_peerworkflow.item_id, openedx.assessment_staffworkflow.item_id and openedx.assessment_studenttrainingworkflow.item_id.
2. Create a new course with an ID > 70 characters.
3. Create an Open Response component that has Student training and Peer and Staff steps.
4. Submit responses and assess them.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
